client/my-sites/upgrades/checkout-thank-you/header/style.scss

```diff
@@ -0,0 +1,71 @@
+.checkout-thank-you-header {
+	border-bottom: 1px solid lighten( $gray, 30% );
+	padding-bottom: 20px;
+}
+
+.checkout-thank-you-header__icon {
+	display: inline-block;
+	position: relative;
+
+	&::before {
+		@include noticon( '\f425', 70px );
+		color: lighten( $gray, 20% );
+	}
+
+	&::after {
+		@include noticon( '\f418', 18px );
+		background: #53b66c;
+		border-radius: 18px;
+		color: $white;
+		position: absolute;
+		top: 6px;
+		right: 15px;
+		width: 18px;
+		height: 18px;
+	}
+}
+
+.checkout-thank-you-header__heading {
+	font-size: 25px;
+	font-weight: 100;
+	margin: 0;
+}
+
+.checkout-thank-you-header__text {
+	font-size: 20px;
+	font-weight: 200;
+	opacity: 0.6;
+	margin: auto;
+	max-width: 50%;
+}
+
+.checkout-thank-you-header__heading,
+.checkout-thank-you-header__text {
+	color: $gray-dark;
+
+	@include breakpoint( "<660px" ) {
+		font-size: 17px;
+		max-width: 100%;
+	}
+}
+
+.checkout-thank-you {
```

Comment:
Because the component itself has access to a flag that determines whether it is loading or not, I think we should apply an `.is-placeholder` class to `div.checkout-thank-you-header` in its render() and change this selector to `.checkout-thank-you-header.is-placeholder`.

I don't think this falls under the umbrella of the CSS issues that @scruffian mentioned as being subject to change from other issues – apologies if that is not correct.